pkg/sql/show.go

```diff
@@ -857,6 +860,191 @@ func (p *planner) ShowConstraints(
 	}, nil
 }
 
+func (p *planner) ShowQueries(ctx context.Context, n *parser.ShowQueries) (planNode, error) {
```

Comment:
@itsbilal, based on my testing, no privileges are required for `SHOW SESSIONS`. Is that correct/intended? If so, it might be a good idea to include a comment here, as we do for most other statements, e.g, https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/show.go#L486.